recipes-kcalendarcore/kcalendarcore/kcalendarcore_git.bb

```diff
@@ -0,0 +1,21 @@
+SUMMARY = "KCalendarCore calendar library"
```

Comment:
Also I see you put this in a top level recipes-kcalendarcore but this seems extremely specific, we tend to put recipes in much broader categories, how about putting it in `recipes-qt` next to kcalcore https://github.com/AsteroidOS/meta-asteroid/blob/d2b91f92c969b2201d896ef1d809efc9510ea82c/recipes-qt/qt5/kcalcore_git.bb ?

Reply:
done!